setup.cfg

```diff
@@ -22,3 +22,6 @@ universal=1
 # http://pep8.readthedocs.org/en/latest/intro.html#error-codes
 # E402: module level import not at top of file
 ignore=E402
+
+[coverage:run]
+omit=nilearn/externals/*
```

Comment:
The documentation showed this with a space before and presumably after the `=` . Dunno if that makes any difference, but maybe try that? Also `./nilearn/externals/*` might work, though the doc doesn't say so, but it sometimes does.

Reply:
Irrelevant,